src/main/java/com/twilio/http/HttpClient.java

```diff
@@ -12,6 +12,9 @@
     public static final int RETRIES = 3;
     public static final long DELAY_MILLIS = 100L;
 
+    public Response lastResponse;
```

Comment:
Should either be private (can only be accessed in the class), or protected (can only be access by class and its subclasses).